### ISSUE TITLE
Numeric decimal dropped when saving multiple worksheets to the same woorkbook

### ISSUE THREAD

**Issue by matsovef** [dfinke/ImportExcel#198]
I've a PowerShell script that first creates a worksheet in a new workbook (refered to as destination hereafter) and then in a loop creates a new worksheet in a new workbook (refered to as source hereafter) that is added to destination as a new worksheet using Copy-ExcelWoorkSheet.

When opening destination in Excel all numeric values in worksheets originating form source has their decimal dropped. For instance the cell value 123.456 in source gets the cell value 123456 in the destination work sheet.

I've debugged my script and confirmed that source (opened in Excel) has numeric decimal values before being copied to destination. The original worksheet in destination isn't affected.

**Comment by dfinke** [comment]
Please post examples, it'll be easier to help.

**Comment by matsovef** [comment]
I've attached the first file, the second file and the result of the second file copied into the first file (combined). As you can see the numeric values in the second file have their decimal point dropped in the second worksheet of combined file and hence are much larger numbers.

I use the following command.
Copy-ExcelWorkSheet `
  -SourceWorkbook .\Second.xlsx `
  -SourceWorkSheet "SCR 9388" `
  -DestinationWorkbook .\First.xlsx

[Combined.xlsx](https://github.com/dfinke/ImportExcel/files/1130344/Combined.xlsx)
[Second.xlsx](https://github.com/dfinke/ImportExcel/files/1130346/Second.xlsx)
[First.xlsx](https://github.com/dfinke/ImportExcel/files/1130345/First.xlsx)

**Comment by DarkLite1** [comment]
In `Copy-ExcelWorkSheet` the following code is used:

`Import-Excel -Path $SourceWorkbook -WorkSheetname $SourceWorkSheet`

In my latest pull request I have redesigned `Import-Excel` to be more robust and feature rich. There's also a new function in there called `Add-Property`. This function is there on purpose so we can fix issues like these like we do in `Export-Excel`. 

The only thing we need to do is add the logic from `Export-Excel` of the function `Add-CellValue` to the function `Add-Property` in the new `Import-Excel`.

**Comment by matsovef** [comment]
After some debugging I found out what is the problem but not how to solve it.

I debugged the last version and in Export-Excel.ps1 we have the following code:
``

**Comment by matsovef** [comment]
After some debugging I found out what is the problem but not how to solve it.

In Export-Excel.ps1 we have the following code:
``
Default {
    #region Save a value as a number if possible
    if (($Number = ConvertTo-Number $_) -ne $null) {
        $TargetCell.Value = $Number
        $targetCell.Style.Numberformat.Format = $Numberformat
        Write-Verbose "Cell '$Row`:$ColumnIndex' header '$Name' add value '$($TargetCell.Value)' as number converted from '$_' with format '$Numberformat'"
    }

**Comment by matsovef** [comment]
After some debugging I found out what is the problem.

In Export-Excel starting with line 464 (version 5.0.0) we find the following lines:

`Default {`
`    #region Save a value as a number if possible`
`    if (($Number = ConvertTo-Number $_) -ne $null) {`
`        $TargetCell.Value = $Number`
`        $targetCell.Style.Numberformat.Format = $Numberformat`

The strange thing here is that (according to debugger) `$_ = 301034,845381085` while the `$Value` parameter in `ConvertTo-Number` has the value `301034.845381085`. In the parameter the `NumberDecimalSeparator` has changed from a decimal comma to a decimal point.

In the ConvertTo-Number function the `[System.Globalization.NumberFormatInfo]::CurrentInfo` defines the `NumberDecimalSeparator` as `,`. When parsing the parameter value the `[Double]::TryParse` method ignores the decimal point and returns `301034845381085` as value.

I noticed that `ConvertTo-Number` declares the `$Value` parameter as `[String]`. A tried two variants in the debugger with the following result:
`$_.ToString()` results in `"301034,845381085"`
`[String]$_` results in `"301034.845381085"`
`$_.GetType()` results in `Double`

It seems that casting a `Double` to `String` causes it to be converted into a string format according to PowerShell default (using a decimal point) while the `ToString` method uses the CultureInfo to format the decimal value as a string in accordance with the current culture.

I guess the fix here is to use the `ToString()` method on `$_` before passing it to `ConvertTo-Number` since it is expecting a string parameter.
